index.html

```diff
@@ -0,0 +1,571 @@
+  @coreSchema(feature: "https://example.com/example/v1.0")
+{
+  query: Query
+}
+
+type SomeType {
+  field: Int @example
+}
+
+directive @example on FIELD
+```
+
+## `@core`
+
+```graphql
+#::[The `@core` directive]
+directive @core(feature: core__FeatureUrl!, as: String, export: Boolean)
+  repeatable on SCHEMA
+```
+
+Declare a core feature present in this schema.
+
+Documents **must** include a definition for the `@core` directive. The provided definition must be *compatible* with the definition above, but may:
+- **Omit optional arguments** if they are never used in the document,
+- **Omit locations** where the directive never occurs,
+- **Introduce new directive locations**, the behavior of which is unspecified here.
+- **Introduce new arguments.** New arguments **must** be prefixed, e.g. `example__extensionArgument: Bool`. The prefix **must** be the name of a feature referenced with this or another `@core` directive within the document.
+- **Use `String` in place of custom scalars.** This is an ease-of-authoring affordance which allows document authors to omit the definitions of custom scalar types provided by features. Processors **must** continue to interpret such arguments and fields according to the encoding and decoding rules of the custom scalar type. For example, the `@core` directive may be defined as follows, with no change to the behavior:
+
+```graphql
+#::[The `@core` directive, defined with `feature: String`]
+directive @core(feature: String)
+```
+
+### `feature: `[core__FeatureUrl](#core__featureurl)
+
+A [feature URL](#core__FeatureUrl) specifying the directive and associated schema elements. When viewed, the URL **should** provide the content of the appropriate version of the specification in some human-readable form. In short, a human reader should be able to click the link and go to the docs for the version in use. There are specific requirements on the format of the URL, but it is not required that the *content* be machine-readable in any particular way.
+
+<figure>
+  <figcaption>
+    <a name=anatomy-of-a-feature-url href=#anatomy-of-a-feature-url class=a-header>Basic anatomy of a feature URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>http://spec.example.com/a/b/c/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+  </code>
+</figure>
+
+Feature URLs contain information about the spec's [prefix](#prefixing) and [version](#versioning), and should be generated and processed in accordance with the [requirements on the structure of feature URLs](#feature-urls).
+
+### `as: String`
+
+Change the [names](#prefixing) of directives and schema elements from this specification. The specified string **must** be a valid GraphQL identifier and **must not** contain the namespace separator (two underscores, `__`).
```

Comment:
I think it should also not end with an underscore, since swapping in `foo_` for `bar` gives you names like `foo___x` which will be parsed as `foo/_x` rather than `foo_/x`, no?